extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -161,6 +161,13 @@ require([
 			event.preventDefault();
 		});
 
+		$('.embeddable-discussions-module').on('click', '.embeddable-discussions-side-spaced', function(event) {
+			if (!$(event.target).is('.avatar-username')) {
+				window.location.href = event.currentTarget.getAttribute('data-url');
```

Comment:
This is not safe to get the unvalidated URL from a HTML attribute that a user can fake. This as coded right now can be used to allow an attacker to make every link on a wiki page force redirect the user to an attack site when clicked just by editing the page. I can provide a PoC of this attack if you're interested. But in general, don't trust anything from the DOM, especially given we're a UGC site.